starport/cmd/network_campaign_list.go

```diff
@@ -0,0 +1,76 @@
+package starportcmd
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/tendermint/starport/starport/pkg/cosmosaccount"
+	"github.com/tendermint/starport/starport/pkg/entrywriter"
+	"github.com/tendermint/starport/starport/services/network/networktypes"
+)
+
+var CampaignSummaryHeader = []string{
+	"id",
+	"name",
+	"coordinator id",
+	"mainnet id",
+}
+
+// NewNetworkCampaignList returns a new command to list all published campaigns on Starport Network
```

Comment:
```suggestion
// NewNetworkCampaignList returns a new command to list all published campaigns on Starport Network.
```

Reply:
we use a period (.) after all other command descriptions, so let's be consistent here.

@Pantani can you help me understand what "returns a new command"? 
or does the NewNetworkCampaignList command "return a list of the campaigns that are published on Starport Network"

Reply:
Returns a new command means the method returns a new CLI command object. When do you type `starport -h`, all commands generated from this methods are there